src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -30,6 +30,15 @@ public unsafe partial class RaspberryPi3Driver : GpioDriver
         private UnixDriver _sysFSDriver = null;
         private readonly IDictionary<int, PinMode> _sysFSModes = new Dictionary<int, PinMode>();
 
+        /// <summary>
+        /// Returns true if this is a Raspberry Pi4
+        /// </summary>
+        protected bool IsPi4
```

Comment:
I'd rather create an enum for this, I'm expecting there will be new version eventually and I don't necessarily want us to have `IsPi4` .. `IsPi15`. Also might be easier to first discuss these changes in the issue

Reply:
This PR is considered a quick fix, because I consider it a quite critical issue, making the whole library use problematic on the Pi4, so I just used the fastest approach.

Reply:
public/protected members in this library will need to be supported forever. This will likely get forgotten if we don't fix right now